Documentation incomplete: Return code 3 is missing

### Type of issue

Missing information

### Description

The article states that there are the return codes 0, 1 and 2 for signtool.exe.
However, when using the official Azure Task "TrustedSigning@0", the pipeline fails with return code 3:

```
Starting: Sign with Trusted Signing
==============================================================================
Task         : Trusted Signing
Description  : This task enables users to sign their files with the Trusted Signing service.
Version      : 0.5.9
Author       : Microsoft
Help         : 
==============================================================================
Checking for required dependencies.
	Build tools package installed: True
	Trusted signing package installed: True
	Sign CLI package installed: True
	All required dependencies are installed.

Creating metadata file: C:\Users\<AGENT USER>\AppData\Local\TrustedSigning\Microsoft.Trusted.Signing.Client\Microsoft.Trusted.Signing.Client.1.0.95\bin\x64\metadata.json

Getting the list of files to be signed.
	No files list was provided.
	Listed files: 0

	Getting files from folder: D:\agent\_work\3\s\Setup\<PATH TO SETUP DIRECTORY>\
	Filter: exe
	Filtered files: 1
		D:\agent\_work\3\s\Setup\<PATH TO SETUP FILE>
	No catalog file was provided.
	Catalog files: 0

Formatting the list of files to be signed.

Batching the list of files to be signed by SignTool.
	Batched SignTool files: 1

Batching the list of files to be signed by SignCli.
	No SignCli files were found to sign.

Signing SignTool file batch 1 of 1.
	Executing signtool.exe: C:\Users\<AGENT USER>\AppData\Local\TrustedSigning\Microsoft.Windows.SDK.BuildTools\Microsoft.Windows.SDK.BuildTools.10.0.26100.4188\bin\10.0.26100.0\x64\signtool.exe sign /v /debug /fd SHA256 /tr http://timestamp.acs.microsoft.com/ /td SHA256 /dlib "C:\Users\<AGENT USER>\AppData\Local\TrustedSigning\Microsoft.Trusted.Signing.Client\Microsoft.Trusted.Signing.Client.1.0.95\bin\x64\Azure.CodeSigning.Dlib.dll" /dmdf "C:\Users\<AGENT USER>\AppData\Local\TrustedSigning\Microsoft.Trusted.Signing.Client\Microsoft.Trusted.Signing.Client.1.0.95\bin\x64\metadata.json" "D:\agent\_work\3\s\Setup\<PATH TO SETUP FILE>"
##[error]SignTool failed with exit code 3
Finishing: Sign with Trusted Signing
```


### Page URL

https://learn.microsoft.com/en-us/dotnet/framework/tools/signtool-exe#return-value

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/framework/tools/signtool-exe.md

### Document Version Independent Id

9ea61916-a9db-f23e-4c43-edc06650483d

### Platform Id

17012f2a-06ef-a445-76dc-ed91d5b09adc

### Article author

@gewarren

### Metadata

* ID: 72a4dbac-a145-e79f-121d-3d8df67552b1
* PlatformId: 17012f2a-06ef-a445-76dc-ed91d5b09adc 
* Service: **dotnet-framework**

--- a/docs/framework/tools/signtool-exe.md
+++ b/docs/framework/tools/signtool-exe.md
@@ -153,6 +153,7 @@ signtool [command] [options] [file_name | ...]
 |0|Execution was successful.|  
 |1|Execution has failed.|  
 |2|Execution has completed with warnings.|  
+|3|Execution failed with RuntimeException.|
 
 ## Examples  
 

